docs/sql-ref-syntax-qry-select-window.md

```diff
@@ -94,107 +94,112 @@ INSERT INTO employees VALUES ("Paul", "Engineering", 29000, 23);
 INSERT INTO employees VALUES ("Chloe", "Engineering", 23000, 25);
 
 SELECT * FROM employees;
-+-----+-----------+------+-----+
-| name|       dept|salary|  age|
-+-----+-----------+------+-----+
-|Chloe|Engineering| 23000|   25|
-| Fred|Engineering| 21000|   28|
-| Paul|Engineering| 29000|   23|
-|Helen|  Marketing| 29000|   40|
-|  Tom|Engineering| 23000|   33|
-| Jane|  Marketing| 29000|   28|
-| Jeff|  Marketing| 35000|   38|
-| Evan|      Sales| 32000|   38|
-| Lisa|      Sales| 10000|   35|
-| Alex|      Sales| 30000|   33|
-+-----+-----------+------+-----+
++-----+-----------+------+---+
+| name|       dept|salary|age|
++-----+-----------+------+---+
+|Chloe|Engineering| 23000| 25|
+| Paul|Engineering| 29000| 23|
```

Comment:
Why are these changed?

Reply:
Update the result with spark 4.0.0-preview1